project/src/index.tsx

```diff
@@ -2,12 +2,22 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './components/app/app';
 
+const Data = {
+  title: 'The Grand Budapest Hotel',
+  genre: 'Drama',
+  date: 2014
+} as const;
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <App
+      title = {Data.title}
```

Comment:
Форматируем только так
```suggestion
      title={Data.title}
```